packages/forms/backend/sirius-components-collaborative-widget-reference/src/main/java/org/eclipse/sirius/components/collaborative/widget/reference/handlers/CreateElementEventHandler.java

```diff
@@ -87,8 +95,14 @@ public void handle(One<IPayload> payloadSink, Many<ChangeDescription> changeDesc
             if (optionalWidget.isPresent() && optionalWidget.get().isReadOnly()) {
                 payload = new ErrorPayload(input.id(), this.messageService.unableToEditReadOnlyWidget());
             } else {
-                var handlerInput = this.getHandlerInput(editingContext, input);
-                Optional<Object> optionalObject = optionalWidget.map(ReferenceWidget::getCreateElementHandler).map(handler -> handler.apply(handlerInput));
+                Optional<Object> optionalObject = this.referenceWidgetCreateElementHandlers.stream()
+                        .filter(provider -> provider.canHandle(input.descriptionId()))
+                        .findFirst()
```

Comment:
The default `ReferenceWidgetDefaultCreateElementHandler` will always return `true` to `canHandle`. How can we be sure it's not the one we get here instead of another, application-provided one?

Reply:
I changed the approch, now I explicitly call this service when no other canHandle is true.
This way, an application that wants to provide its own handler, doesn't need to play with spring annotation @Order